### ISSUE TITLE
Change BitBucket Server URL

### ISSUE THREAD

**Issue by mikesmithphl** [mark-adams/open-on-bitbucket#7]
Is it currently possible to change what url the file will be opened to? Looks like it defaults to `stash`.

**Comment by jnak** [comment]
+1 
Description seems to suggest we can point it to our own hosted stash / bitbucket server for on-premise installs

**Comment by dykim78** [comment]
Yep, we'd like to change the own/local bitbucket server instead of default. Is it possible to change the target server?

**Comment by michaelsanford** [comment]
It seems to try to detect the host from `git remote`, [from the lib](https://github.com/mark-adams/open-on-bitbucket/blob/master/lib/bitbucket-file.coffee#L169) and below.

...notwithstanding #9 (which is a problem).
